### PR TITLE
feat(metrics): expose workspace reliability summary

### PR DESCRIPTION
**Pull request by dimileeh**
Automatically opened by AWF workspace `ws_610e40e03b62476eac65f855` (agent: `codex`).

**External task ID**: awf-wave3-metrics-summary-20260426T055128Z

### Task
Implement a small PRD observability slice: a workspace reliability summary API that a console can use for operational health cards.

Required behavior:
- Add `GET /v1/metrics/workspaces/summary`.
- Response should include counts by workspace status, counts by failure_reason, active workspace count, completed count, failed count, cancelled count, destroyed count, cleanup_failure count, and simple recent-window support via `since_hours` query param defaulting to 24 and bounded 1..168.
- Include a `generated_at` timestamp and `window_start` timestamp.
- Keep this read-only and token-free for now, matching existing non-sensitive list APIs.
- Use a new `awf.service.metrics` module; avoid changing broad repository code unless absolutely necessary.

TDD requirements:
- Add failing service and API tests first for empty DB, mixed statuses, failure reasons, since_hours filtering, and query validation.

Validation commands:
- uv run ruff check src/awf/api/routes/metrics.py src/awf/api/app.py src/awf/service/metrics.py tests/unit/api/test_metrics.py tests/unit/service/test_metrics.py
- uv run mypy src/awf/api/routes/metrics.py src/awf/api/app.py src/awf/service/metrics.py
- uv run pytest tests/unit/api/test_metrics.py tests/unit/service/test_metrics.py -q

Strict TDD and AWF contract:
- Write focused failing tests first and commit the red tests, including the failure summary in the commit body.
- Implement until green, then run the validation commands listed below.
- Commit locally with conventional commits.
- Do not push, switch branches, rebase, open PRs, merge PRs, or resolve GitHub threads manually outside the task contract. AWF owns those lifecycle steps.
- Keep the change tightly scoped to the owned paths.

---
Validation: 6 profile command(s) passed inside the workspace container.

### CODE REVIEW

File: src/awf/service/metrics.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+    completed_count = status_counts[WorkspaceStatus.completed.value]
+    failed_count = status_counts[WorkspaceStatus.failed.value]
+    cancelled_count = status_counts[WorkspaceStatus.cancelled.value]
+    destroyed_count = status_counts[WorkspaceStatus.destroyed.value]
+    active_count = sum(
+        count
+        for status, count in status_counts.items()
+        if status not in TERMINAL_WORKSPACE_STATUSES
+    )
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/service/metrics.py]: fixed in commit 48c9dd3242a80b71104e319210e3bcb1fc153cb5

File: src/awf/service/metrics.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+MAX_SUMMARY_WINDOW_HOURS = 168
+
+TERMINAL_WORKSPACE_STATUSES = frozenset(
+    {
+        WorkspaceStatus.completed.value,
+        WorkspaceStatus.failed.value,
+        WorkspaceStatus.cancelled.value,
+        WorkspaceStatus.destroyed.value,
+    }
+)
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/service/metrics.py:25]: fixed in commit 478520ece0ab5fe12c7ce019ed4fad729f293d9c

File: src/awf/service/worker.py [hunk 3]
```diff
@@ -94,6 +94,10 @@ def _pr_monitor_factory(
         profile: WorkspaceProfile,
         workspace: Workspace,
     ) -> Any:
+        nonlocal merge_coordinator
+        if merge_coordinator is None:
+            merge_coordinator = _merge_coordinator_for_database_url(settings.database_url, engine=engine)
```
[the remark this answers is not in the document]
Comment (dimileeh) [reply src/awf/service/worker.py]: fixed in commit dd22400521f1d7ee4693f3e3b8bcac258ecbcd53
